examples/simple/igraph_get_all_shortest_paths_dijkstra.c

```diff
@@ -149,6 +157,20 @@ int main() {
         VECTOR(res)[i] = 0;
     }
 
+
```

Comment:
Add a comment here explaining that we are testing the combination of `res = 0` and `res_e != 0` here.